src/pages/signup-affiliates/components/_account-details.tsx

```diff
@@ -101,8 +167,56 @@ const AccountDetails = () => {
             label: localize('Postal/Zip code'),
             placeholder: 'Postal/Zip code',
             required: true,
+            error: postcode_error_msg,
+            value: postal_code,
         },
     ]
+    const handleInput = (e) => {
+        const { name, value } = e.target
+        switch (name) {
+            case 'country': {
+                setCountry(value)
+                return setCountryErrorMsg(validateCountry(value))
+            }
+            case 'state': {
+                setState(value)
+                return setStateErrorMsg(validateState(value))
+            }
+            case 'city': {
+                setCity(value)
+                return setCityErrorMsg(validateCity(value))
+            }
+            case 'street': {
+                setStreet(value)
+                return setStreetErrorMsg(validateStreet(value))
+            }
+            case 'postal_code': {
+                setPostCode(value)
+                return setPostCodeErrorMsg(validatePostCode(value))
+            }
+        }
+    }
+
+    const validateCountry = (country_str) => {
+        const error_message = validation.country(country_str)
+        return error_message
+    }
+    const validateState = (state_str) => {
+        const error_message = validation.address_state(state_str)
+        return error_message
+    }
```

Comment:
No need to create the function here just use it directly in `handleInput` function